cmd/seid/cmd/root.go

```diff
@@ -126,20 +133,27 @@ func initRootCmd(
 	)
 
 	// add server commands
-	server.AddCommands(
```

Comment:
do we also need the `sdkserver.AddCommands` here to keep seid command working

Reply:
nope, basically ethermintserver.AddCommands is a superset of server.AddCommands, but includes necessary ones for evm. See https://github.com/evmos/ethermint/blob/main/server/util.go#L24
Note that it has `sdkserver` which is what we have here as `server`